ports/gdal/vcpkg.json

```
@@ -39,7 +39,33 @@
     "zlib"
   ],
   "default-features": [
-    "default-features"
+    "gif",
+    {
+      "name": "hdf5",
+      "platform": "!uwp & !(windows & arm64) & !android & !ios"
```

```suggestion
      "platform": "!uwp"
```
Port hdf5 supports cross-builds now.